.travis.yml

```diff
@@ -27,9 +31,20 @@ env:
     - WP_VERSION=4.7 WP_MULTISITE=1
 
 before_script:
+    - |
+      if [ -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini ]; then
+        phpenv config-rm xdebug.ini
+      else
+        echo "xdebug.ini does not exist"
+      fi
     - bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
-
-#after_success:
-#    - bash <(curl -s https://codecov.io/bash)
```

Comment:
What's about this line?

Reply:
I've removed those lines just to clean up the file. They are commented out anyway so wasn't being used.

If you really want me to add the commented lines back in I can, although I don't know why it would be needed?